package.json

```diff
@@ -40,6 +44,9 @@
     "long-timeout": "0.1.1",
     "sorted-array-functions": "^1.3.0"
   },
+  "optionalDependencies": {
```

Comment:
I don't think this will work correctly as npm still tries to install optionalDependency. It should be an optional peerDependency, see https://github.com/knex/knex/blob/master/package.json for an example

Reply:
I would tend to disagree, that as an optional dependency the user should have to exclude it via npm --no-optional.
I am happy to rework this to meet your standards as you are the maintainer, but personally believe that it should be listed as an optional dependency rather than a peer dependency.

Reply:
Since this is a **very** heavyweight dependency (3 Mb unpacked) I don't feel comfortable including it by default, especially since browsers are one of the supported environments, and size of the bundle absolutely counts there. Which is why it should be opt-in and not opt-out.